science/parallel-netcdf/Portfile

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               mpi 1.0
+PortGroup               active_variants 1.1
+
+compilers.choose        f90 cxx
+
+mpi.setup
+
+name                    parallel-netcdf
+version                 1.10.0
+
+maintainers             {@thiagoveloso gmail.com:thiagoveloso} openmaintainer
+platforms               darwin
+categories              science devel
+license                 Permissive
+
+description             A Parallel I/O Library for NetCDF File Access.
+
+long_description        PnetCDF is a high-performance parallel I/O library \
+                        for accessing files in format compatibility with Unidata's \
+                        NetCDF, specifically the formats of CDF-1, 2, and 5. The \
+                        CDF-5 file format, an extension of CDF-2, supports unsigned \
+                        data types and uses 64-bit integers to allow users to define \
+                        large dimensions, attributes, and variables (> 2B array elements).
+
+homepage                https://parallel-netcdf.github.io/
+
+master_sites            http://cucis.ece.northwestern.edu/projects/PnetCDF/Release/
+
+checksums               rmd160  4de992174820518019392150d976d7f7eb81375e \
+                        sha256  ed189228b933cfeac3b7b4f8944eb00e4ff2b72cf143365b1a77890980663a09 \
+                        size 	2089202
+
+compilers.setup         default
+
+mpi.setup               require
+
+
+depends_lib             port:autoconf \
+                        port:automake \
```

Comment:
When using portgroups, use `depends_lib-append` instead to not overwrite the portgroup dependencies. 

Also, usually autoconf and automake are build dependencies.